catroidTest/src/org/catrobat/catroid/uitest/content/brick/PlaceAtBrickTest.java

```diff
@@ -94,6 +95,26 @@ public void testPlaceAtBrick() throws InterruptedException {
 				R.id.brick_place_at_edit_text_y, yPosition, Brick.BrickField.Y_POSITION, placeAtBrick);
 	}
 
+	public void testBehaviorOfUndoAndRedoButton(){
+		solo.clickOnText("105");
+		solo.clickOnButton("5");
```

Comment:
Use the ID of the button rather than the text. You can find it in `formula_editor_keyboard.xml`